app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __repr__(self) -> str:
+        return (f"{{"
```

Comment:
It's better to use __str__ when you want to give a name for an object. __repr__ is mostly used to see detailed name of object when you work from console(like memory cell, type, etc)